f3read.c

```diff
@@ -386,11 +386,13 @@ static void iterate_files(const char *path, const long *files,
 		tot_overwritten += stats.secs_overwritten;
 		tot_size += stats.bytes_read;
 		and_read_all = and_read_all && stats.read_all;
-		if (stats.secs_corrupted && delete) {
+		if (delete && (stats.secs_corrupted || tot_changed || tot_overwritten)) {
```

Comment:
Please replace `tot_changed` with `stats.secs_changed` and `tot_overwritten` with `stats.secs_overwritten`. My fault, again.

Break the line as needed to fit within the 80-column limit.